site.yml

```diff
@@ -1,7 +1,13 @@
 ---
-- hosts: localhost
+- name: Run Default Installation Plays
+  hosts: localhost
   gather_facts: true
   tasks:
+    - name: Checking for Pre-installation Ansible Tasks
+      include_tasks: "{{lookup('first_found', ansible_pre_tasks)}}"
```

Comment:
I'm a short on Ansible knowledge, so pardon if I'm missed something, but from basic googleing, it doesn't appear that this will work with a URL. If we're going to avoid creating our own image, then this probably will need to support that.

Reply:
This looks for the file on disk, if we want to handle url, I can add a download task for it.

Reply:
I expected people to mount in the volume their own custom tasks to extend, making it function like the others seems like a good idea.

Reply:
Using `url_get` works either way, as it supports `file:///`, that's what I'd recommend. Again though, I'm an Ansible novice.

Reply:
I realized that in k8s, I can mount these tasks in from a configmap. I'm playing with that now, but it should work without issue.

Reply:
added in some download tasks, going to make it take a list and should function well.

Reply:
K think this is ready for re-review